jboss-eap-config-tracing/module.yaml

```diff
@@ -0,0 +1,17 @@
+schema_version: 1
+name: jboss.eap.config.tracing
+version: '1.0'
+description: EAP OpenTracing configuration scripts
+execute:
+- script: configure.sh
+  user: '185'
+envs:
+    - name: "WILDFLY_TRACING_ENABLED"
+      example: "true"
+      description: "Use the microprofile-tracing-smallrye subsystem to add OpenTracing support."
+    - name: "JAEGER_AGENT_HOST"
+      example: "true"
```

Comment:
"true" does not make sense as en example here.

Reply:
Oops; thanks, will fix.

Reply:
I replaced this with an example hostname from a Service that exposes a jaeger agent.